### PR TITLE
Docs build

### PR DESCRIPTION
**Pull request by direnakkocdemir**

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: docs-build
+
+on:
+  push:
+    branches: [ docs-build ]
```
Comment (eoinsha) [inline .github/workflows/docs.yml:8]: Can we change this to `main` before merge?

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          npm ci
+          else
+          npm i
+          fi
+          npm run build
+      env:
+        CI: true   
+
+    - name: Release to GH pages if on docs-build
+      if: github.ref == 'refs/heads/docs-build'
```
Comment (eoinsha) [inline .github/workflows/docs.yml:45]: The ref will also have to be specified as `main`, I guess.

File: app.yml [hunk 3]
```diff
@@ -2,14 +2,14 @@
 # SLIC Starter-based application deployment to work
 sourceRepo:
   # This should be changed to match your own repo and branch
-  owner: fourTheorem
+  owner: direnakkocdemir
```
Comment (eoinsha) [inline app.yml:5]: Changes to app.yml aren't required for this PR so we can revert to the defaults

File: docs-build/docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+// baseUrl: the name of the repository
+// url: after your githup name .github.io
+// github user or organization that own the repository
+// projectName: the name of the githuo repository
+// please see the link: https://docusaurus.io/docs/deployment
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'SLIC Starter',
+  tagline: 'SLIC Starter is a complete starter project for production-grade serverless applications on AWS.',
+  url: 'https://direnakkocdemir.github.io',
```
Comment (eoinsha) [inline docs-build/docusaurus.config.js:15]: URL and repo references in this file should be changed to the main slic-starter repo

**Review by eoinsha — CHANGES_REQUESTED** [review]
This  looks really great. We just have to tidy up some of the references to make sure we use the right branch and repo, and revert changes to app.yml - see the specific comments for details!

File: .github/workflows/docs.yml (hunk 1 above)
Comment (direnakkocdemir) [reply .github/workflows/docs.yml:8]: Yes, we can. When we decide that we don't need anymore the docs-build branch, we can modify the docs.yml as it is required.

File: .github/workflows/docs.yml (hunk 2 above)
Comment (direnakkocdemir) [reply .github/workflows/docs.yml:45]: Yes, exactly. I has specified as **main**.

**Comment by alokt** [comment]
Hello guys. Not sure how I'm added to this repo. I'm not supposed to be part of this dev :)

**Review by eoinsha — APPROVED** [review]
LGTM. Thanks a million!

**Pull request merged**